### PR TITLE
Find dependencies when including tmxparserConfig.cmake

### PR DESCRIPTION
**Pull request by sjdrc**
Hey, I missed modifying this file when hunterising this package. Let me know if you want me to do a rebase to clean up the history.

### CODE REVIEW

File: cmake/tmxparserConfig.cmake.in [hunk 1]
```diff
@@ -1 +1,6 @@
-include("${CMAKE_CURRENT_LIST_DIR}/tmxparserTargets.cmake")
\ No newline at end of file
+include(CMakeFindDependencyMacro)
+
+find_dependency(tinyxml2 REQUIRED)
+find_dependency(ZLIB REQUIRED)
```
Comment (ruslo) [inline cmake/tmxparserConfig.cmake.in]: We need `CONFIG` here to not to fall into system version:
- https://docs.hunter.sh/en/latest/packages/pkg/ZLIB.html

File: cmake/tmxparserConfig.cmake.in [hunk 2]
```diff
@@ -1 +1,6 @@
-include("${CMAKE_CURRENT_LIST_DIR}/tmxparserTargets.cmake")
\ No newline at end of file
+include(CMakeFindDependencyMacro)
+
+find_dependency(tinyxml2 REQUIRED)
```
Comment (ruslo) [inline cmake/tmxparserConfig.cmake.in]: Same:
- https://docs.hunter.sh/en/latest/packages/pkg/tinyxml2.html

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by sjdrc** [comment]
@ruslo I think this is ready to be merged

**Comment by ruslo** [comment]
Released: https://github.com/hunter-packages/tmxparser/releases/tag/v2.1.0-ab4125b-p1

**Pull request merged**